conf/ocsci/fips.yaml

```
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  fips: 'true'
```

```suggestion
  fips: true
```
use boolean value and not string

---

this is what I tried before and ended up the upper case with fips: True 
See the previous commit